src/java.management.rmi/share/classes/javax/management/remote/rmi/RMIConnectorServer.java

```diff
@@ -116,7 +116,9 @@ public class RMIConnectorServer extends JMXConnectorServer {
     * {@linkplain RMIServer} implementation.
     * If the attribute is not set then any class is deemed acceptable.
     * @see ObjectInputFilter
-    */
+     *
+     * @since 10
+     */
```

Comment:
Please fix the indentation of the "*" for the this comment on CREDENTIALS_FILTER_PATTERN so it looks consistent.

Reply:
Fixed to be consistent, but the rest of the comment is still indented wrong.
```
/**
* Name of the attribute that specifies ...
* ....

Reply:
(I would have gone the other way and corrected the preceding indentation to the standard).